docker/tif-metadata/run.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
```

Comment:
Perhaps move this whole thing into an `app` directory alongside `requirements.txt`.